src/coffee/runtime.py

```
@@ -0,0 +1,1777 @@
+        self.income_claimed = False
+        self.real_purchase_performed = False
+
+        if not self.open_coffee_shop(card):
+            return False, "未能进入一咖舍界面"
+        if self.pending_supply_error:
+            return False, self.pending_supply_error
+
+        self.income_claimed = self.claim_income_if_present()
+        if self.is_income_report_popup():
+            self.actions.append("income_popup_not_closed_after_claim")
+            return False, "营收报告弹窗未关闭，未进入商品/补货流程"
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)